diff --git a/utils/webview_login.py b/utils/webview_login.py
index 6168842..7403d86 100644
--- a/utils/webview_login.py
+++ b/utils/webview_login.py
@@ -138,7 +138,7 @@ def do_webview_login(callback_sid=None, callback_code=None, user_agent=None):
     logger.info('Opening Epic Games login window...')
     # Open logout URL first to remove existing cookies, then redirect to login.
     window = webview.create_window(f'Legendary {__version__} - Epic Games Account Login',
-                                   url=url, width=768, height=1024, js_api=api)
+                                   url=url, width=1280, height=800, js_api=api)
     api.window = window
     window.events.loaded += api.on_loaded
 
